### PR TITLE
Formatting corrections, added war to Tomcat, changed names assigned t…

### PR DESCRIPTION
**Pull request by johnmccabe**
…o ELK entities, misc other minor updates to advanced-example doc.

Just a basic review of the doc, blueprint deploys as expected (doesn't stop completely however).

### CODE REVIEW

**Comment by johnmccabe** [comment]
FYI @Graeme-Miller

File: docs/guide/yaml/advanced-example.md [hunk 1]
```diff
@@ -74,10 +77,10 @@ customize.command: |
...
 
 ##### Check running
-After an app is installed and run, this command will be run regularly and used to populate the "service.isUp" sensor. If this command is not specified, or returns an exit code of anything other than zero, then Brooklyn will assume that your entity is broken and will display the fire status symbol. Please refer to the following section of the Elasticsearch blueprint:
+After an app is installed and run, this command is scheduled to run regularly and used to populate the `service.isUp` sensor. If this command is not specified, or returns an exit code of anything other than zero, then Brooklyn will assume that your entity is has failed and will display the fire status symbol. Please refer to the following section of the Elasticsearch blueprint:
```
Comment (Graeme-Miller) [inline docs/guide/yaml/advanced-example.md]: 'is has failed',
should be 'is broken' or 'has failed'

File: docs/guide/yaml/example_yaml/brooklyn-elk-catalog.bom [hunk 2]
```diff
@@ -9,27 +9,28 @@ brooklyn.catalog:
...
-        name: kibana
+        name: Kibana Server
         customize.latch: $brooklyn:component("es").attributeWhenReady("service.isUp")
         brooklyn.config:
           kibana.elasticsearch.ip: $brooklyn:component("es").attributeWhenReady("host.address.first")
           kibana.elasticsearch.port: $brooklyn:component("es").config("elasticsearch.http.port")
       - type: org.apache.brooklyn.entity.webapp.tomcat.Tomcat8Server
-        id: jboss
+        id: tomcat
+        war: https://tomcat.apache.org/tomcat-8.0-doc/appdev/sample/sample.war
```
Comment (Graeme-Miller) [inline docs/guide/yaml/example_yaml/brooklyn-elk-catalog.bom]: best to leave out the war as it creates a dependency of that URL. The example works without it (you can access the tomcat admin UI).

**Comment by Graeme-Miller** [comment]
@johnmccabe thanks for looking into this. I have just added two comments.

**Comment by johnmccabe** [comment]
@Graeme-Miller have updated with your comments.

**Comment by ahgittin** [comment]
excellent, merging

**Pull request merged**